pkg/k8sclient/k8sclient.go

```diff
@@ -551,6 +551,12 @@ func getNetDelegate(client *ClientInfo, pod *v1.Pod, netname, confdir, namespace
 			return delegate, resourceMap, nil
 		}
 	} else {
+		// acquire lock to access file
+		if types.ChrootMutex != nil {
+			types.ChrootMutex.Lock()
+			defer types.ChrootMutex.Unlock()
+		}
+
```

Comment:
There are other places which access filesystem, like option2 in the above, should they be protected by this mutex?

Reply:
Good catch. Fixed.